cmd/buildah/addcopy.go

```diff
@@ -10,25 +10,26 @@ var (
 	addFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "name",
-			Usage: "name or ID of the working container",
+			Usage: "name or `ID` of the working container",
```

Comment:
This doesn't italicize the text (it's not markdown) - it marks a word which should be used instead of "value" when listing a command line flag in --help output.